www/docs/intro.md

```diff
@@ -570,6 +571,12 @@ module.exports = {
       // venv: "path/to/venv" <- for env with installed starknet-devnet (created with e.g. `python -m venv path/to/venv`)
       venv: "<VENV_PATH>",
 
+      // use python or rust vm implementation
+      // vmLang: "python" <- use python vm
+      // vmLang: "rust" <- use rust vm (require to install cairo-rs-py to use rust vm)
```

Comment:
It should be noted that rust VM is available out of the box if using dockerized integrated-devnet but that rustc and cairo-rs-py are required if using installed devnet

Reply:
updated